paper/src/main/java/com/denizenscript/denizen/paper/events/BlockPreDispenseScriptEvent.java

```diff
@@ -0,0 +1,71 @@
+package com.denizenscript.denizen.paper.events;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.ItemTag;
+import com.denizenscript.denizen.objects.LocationTag;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import io.papermc.paper.event.block.BlockPreDispenseEvent;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+public class BlockPreDispenseScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // <block> tries to dispense <item>
+    //
+    // @Group Block
+    //
+    // @Location true
+    //
+    // @Cancellable true
+    //
+    // @Triggers before a block dispenses an item. This event fires before the dispenser finishes processing its drop, allowing access to the slot trying to be dispensed and full cancellation of the dispenser's sounds.
```

Comment:
You can have lines below it as part of the meta (e.g. [delta time event](https://github.com/DenizenScript/Denizen-Core/blob/25cf2298fcdcf282757005ec83a637595a6b106a/src/main/java/com/denizenscript/denizencore/events/core/DeltaTimeScriptEvent.java#L18-L21)), probably makes sense to put the explanation bit in a new line.